python/Ganga/Core/MonitoringComponent/Local_GangaMC_Service.py

```diff
@@ -947,11 +947,11 @@ def __defaultActiveBackendsFunc(self):
                 j = stripProxy(self.registry_slice(i))
 
                 job_status = lazyLoadJobStatus(j)
-                backend_obj = lazyLoadJobBackend(j)
-                backend_name = getName(backend_obj)
 
                 if job_status in ['submitted', 'running'] or (j.master and (job_status in ['submitting'])):
                     if self.enabled is True and self.alive is True:
+                        backend_obj = lazyLoadJobBackend(j)
```

Comment:
If the problem is that creating this (unwanted?) object takes so long would it be better to avoid creating it here at all, we could quite easily add a ```lazyLoadJobBackendName``` method which bypasses the constructor here or is the overhead in this case 'acceptible'?